src/pages/SignIn/index.js

```diff
@@ -22,28 +22,30 @@ export default function SignIn() {
 
   const { eventInfo } = useContext(EventInfoContext);
   const { setUserData } = useContext(UserContext);
-  
+
   function submit(event) {
     event.preventDefault();
     setLoadingSignIn(true);
 
-    api.auth.signIn(email, password).then(response => {
-      setUserData(response.data);
-    }).catch(error => {
-      /* eslint-disable-next-line no-console */
-      console.error(error);
-      
-      if (error.response) {
-        for (const detail of error.response.data.details) {
-          toast(detail);
+    api.auth
+      .signIn(email, password)
+      .then((response) => {
+        setUserData(response.data);

```

Comment:
É interessante manter o código indentado

Reply:
Isso foi a correção de um bug e o slint permitiu, então acredito q esteja correto

Reply:
Ah, eu acho que analisei errado mesmo srsrsrs, pq esse for já foi retirado. Sorry! :back: